ext/c_zookeeper.rb

```diff
@@ -328,7 +328,7 @@ def submit_pending_calls
     end
 
     def wake_event_loop!
-      @pipe_write && @pipe_write.write('1')
+      @pipe_write && !@pipe_write.closed? && @pipe_write.write('1')
```

Comment:
Is this change intentional?

Reply:
This is a fix for a flaky test, eg https://github.com/malmckay/zookeeper/runs/3675238341?check_suite_focus=true